os/include/tinyara/seclink_drv.h

```diff
@@ -15,6 +15,8 @@
  * language governing permissions and limitations under the License.
  *
  ****************************************************************************/
+#pragma once
```

Comment:
why can't we use standard method #ifndef __SECLINK_DRV_H__  ?

Reply:
I think pragma once is easy to read.